### PR TITLE
Admin app list improvements

### PR DESCRIPTION
**Pull request by ryneeverett**
See commit messages for detail.

### CODE REVIEW

**Comment by ryneeverett** [comment]
Heh, might want to change the "COVERAGE DECREASE THRESHOLD FOR FAILURE" setting in coveralls.io from the default of "0".

**Comment by AlexHill** [comment]
@ryneeverett thanks, looks good.

File: mezzanine/core/templatetags/mezzanine_tags.py [hunk 1]
```diff
@@ -559,14 +559,13 @@ def admin_app_list(request):
                 DeprecationWarning)
         in_menu = in_menu and model_admin.has_module_permission(request)
         if in_menu and request.user.has_module_perms(opts.app_label):
-            perms = model_admin.get_model_perms(request)
             admin_url_name = ""
-            if perms["change"]:
+            if model_admin.has_change_permission(request, model):
```
Comment (gradel) [inline mezzanine/core/templatetags/mezzanine_tags.py]: As far as I understand it, has_change_permission takes an object as second argument, not a model. If you want to check permission for all objects of type model, leave out the second argument.

Comment (ryneeverett) [inline mezzanine/core/templatetags/mezzanine_tags.py]: @gradel I think you're right. I did test this and it seems like this should have thrown an error.

Comment (gradel) [inline mezzanine/core/templatetags/mezzanine_tags.py]: @ryneeverett I have django-filer integrated and this is the only app, which really throws an error here.

Comment (ryneeverett) [inline mezzanine/core/templatetags/mezzanine_tags.py]: Oddly enough coveralls says this line runs 6x in tests. https://coveralls.io/builds/6921715/source?filename=mezzanine%2Fcore%2Ftemplatetags%2Fmezzanine_tags.py#L591

Comment (ryneeverett) [inline mezzanine/core/templatetags/mezzanine_tags.py]: I'm guessing that if the user has permissions for all objects of type model, the object is simply ignored. That would explain why I can pass any arbitrary object and it returns `True`.

**Pull request merged**